### PR TITLE
11.15

### PR DESCRIPTION
**Pull request by O-Mr-Wei-O**

### CODE REVIEW

File: wxb/wxb-11.14/Hanoi-module.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var array = [];
+
+module.exports = function hanoi(n, a, b, c) {
+    if (n == 1) {
+        array.push("Move " + n + " from " + a + " to " + c);
```
Comment (calidion) [inline wxb/wxb-11.14/Hanoi-module.js]: push的数据不要转化成字符串

File: wxb/wxb-11.14/Hanoi-module.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    }
+    else {
+        hanoi(n - 1, a, c, b);
+        array.push("Move " + n + " from " + a + " to " + c);
```
Comment (calidion) [inline wxb/wxb-11.14/Hanoi-module.js]: 同上

File: wxb/wxb-11.14/yanghui-module.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+var array1 = new Array();
+module.exports = function yanghui(n) {
+    function array(i, j) {
```
Comment (calidion) [inline wxb/wxb-11.14/yanghui-module.js]: 名字最好修改一下。
array 
=>
add 或者 plus等都是可以的。

File: wxb/wxb-11.14/Hanoi-module.js (hunk 1 above)
Comment (O-Mr-Wei-O) [reply wxb/wxb-11.14/Hanoi-module.js]: [ 'Move 1 from A to C',
  'Move 2 from A to B',
  'Move 1 from C to B',
  'Move 3 from A to C',
  'Move 1 from B to A',
  'Move 2 from B to C',
  'Move 1 from A to C' ]
push存储一句话进数组似乎默认转为String类型的

**Pull request merged**